ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/sflow.schema.yml

```diff
@@ -41,6 +43,9 @@ keys:
                     display_name: Port Number
                     convert_types:
                     - str
+            source:
+              type: str
+              display_name: Source IP
```

Comment:
```suggestion
              description: Source IP Address
```

Reply:
I will also add "Address" to the descriptions of Destination.. to harmonize it